### DISCUSSION TITLE
Fringe pattern in REACH, YJ band

### DISCUSSION THREAD

**Discussion by YuiKasagi** [prvjapan/pyird#53] [Ideas]
In some orders at the YJ band of REACH spectrum, I found a fringe pattern (?) appeared.

It appears in the spectrum extracted from raw data (i.e. without cleaning the readout pattern) and even in the spectrum after cleaning the readout pattern by using pyird. It may be a fringe pattern.
It also appears in the Flat data (of REACH, YJ).

I am not sure, but cannot this be removed by "clean_pattern" #5  in pyird? Should we make another code to treat the fringe (someday)?
![image](https://user-images.githubusercontent.com/66584422/188797414-12512a7c-6e43-407c-b1b1-ed1fa0974b3c.png)

**Comment by HajimeKawahara** [comment]
No, we cannot remove this pattern using `clean_pattern`. `clean_pattern` corrects channel-based systematics. The global pattern cannot be treated. We need to make an additional code. @YuiKasagi

**Reply by YuiKasagi** [reply]
Thank you for the reply! I understand. 
I will consider how to treat it (if I have time). (Fourier transform? or something...)

**Reply by HajimeKawahara** [reply]
Presumably a 2DGP works for that purpose...

**Reply by HajimeKawahara** [reply]
I made a simple one, which works for a large 2d array.
https://github.com/HajimeKawahara/gpkron

**Reply by YuiKasagi** [reply]
That's nice...! I will refer to it.
